rb/lib/selenium/webdriver/chrome/bridge.rb

```diff
@@ -21,88 +21,63 @@ module Selenium
   module WebDriver
     module Chrome
       # @api private
-      class Bridge < Remote::Bridge
-        def initialize(opts = {})
-          port = opts.delete(:port) || Service::DEFAULT_PORT
-          service_args = opts.delete(:service_args) || {}
+      class Bridge < Remote::OSSBridge
+        def driver_extensions
+          [DriverExtensions::TakesScreenshot,
+           DriverExtensions::HasInputDevices,
+           DriverExtensions::HasWebStorage]
+        end
 
-          if opts[:service_log_path]
-            service_args.merge!(service_log_path: opts.delete(:service_log_path))
-          end
+        private
 
-          unless opts.key?(:url)
-            driver_path = opts.delete(:driver_path) || Chrome.driver_path(false)
-            @service = Service.new(driver_path, port, *extract_service_args(service_args))
-            @service.start
-            opts[:url] = @service.uri
-          end
+        def bridge_module
+          Module.nesting[1]
+        end
 
-          opts[:desired_capabilities] = create_capabilities(opts)
+        def process_deprecations(opts)
+          if opts[:service_log_path]
+            opts[:service_args] ||= {}
+            warn <<-DEPRECATE.gsub(/\n +| {2,}/, ' ').freeze
+              [DEPRECATION] Using `:service_log_path` directly is deprecated.  
+              Use `service_args[:service_log_path] = value`, instead.
+            DEPRECATE
 
-          super(opts)
-        end
+            opts[:service_args][:service_log_path] = opts.delete :service_log_path
+          end
 
-        def browser
-          :chrome
-        end
+          [:args, :switches, :detach, :prefs].each do |method|
+            next unless opts.key? method
+            opts[:options] ||= {}
 
-        def driver_extensions
-          [
-            DriverExtensions::TakesScreenshot,
-            DriverExtensions::HasInputDevices,
-            DriverExtensions::HasWebStorage
-          ]
-        end
+            warn <<-DEPRECATE.gsub(/\n +| {2,}/, ' ').freeze
+              [DEPRECATION] Using `#{method}` directly is deprecated.  
+              Use `options[#{method}] = value`, instead.
+            DEPRECATE
 
-        def capabilities
-          @capabilities ||= Remote::Capabilities.chrome
-        end
+            opts[:options][method.to_s] = opts.delete method
+          end
 
-        def quit
           super
-        ensure
-          @service.stop if @service
         end
 
-        private
-
-        def create_capabilities(opts)
-          caps = opts.delete(:desired_capabilities) { Remote::Capabilities.chrome }
+        def process_service_args(service_opts)
+          return [] unless service_opts
+          return service_opts if service_opts.is_a? Array
 
-          chrome_options = caps['chromeOptions'] || caps[:chrome_options] || {}
-          chrome_options['binary'] = Chrome.path if Chrome.path
```

Comment:
`chrome_options['binary']` is no longer being set anywhere, so setting `Chrome.path=` does nothing. Might add it to `Capabilities`